VMBackup/main/blobwriter.py

```diff
@@ -176,7 +178,9 @@ def ClearPageBlob(self, blobUri, blobProperties):
                     http_util = HttpUtil(self.hutil)
                     sasuri_obj = urlparse.urlparse(blobUri + '&comp=page')
                     # Get Blob-properties to know content-length
-                    contentLength = int(blobProperties.contentLength)
```

Comment:
Is there an different handling that is required for blobProperties being NULL instead of letting the flow go through?

Reply:
If blobProperties is null. accessing contentLength property throws exception. Without knowing the contentLenght of the current blob, put_page_clear() from a definitive index is not possible. hence that logic should be skipped which is being taken care by setting the default value of contentLength = 0.